cli/purge.cls.php

```diff
@@ -166,7 +166,30 @@ public function url($args)
 		}
 
 		if (is_multisite()) {
-			if (get_blog_id_from_url($deconstructed['host'], '/') === 0) {
+			$current_blog = get_current_blog_id();
+			$path = '/';
+			// If subfolder install test if we can identify the blog from url.
+			// If subdomain install: $path needs to remain '/'.
+			if(!SUBDOMAIN_INSTALL){
```

Comment:
Directly calling a const is not safe. Please use `is_subdomain_install()`.
If you want to use const, use like how its used in `is_subdomain_install()` defination.

Reply:
Changed